### ISSUE TITLE
Image Block - Include alt text from media library when getting Dynamic Content data

### ISSUE THREAD

**Issue by andeng1106** [gambitph/Stackable#2989]
**Is your feature request related to a problem? Please describe.**
Yes. When I'm fetching a dynamic content from image fields to a stackable image block, it doesn't get the alt text assigned from media library.
- Applies to third party integrations with:
    - ACF
    - JetEngine
    - Meta Box
- Applies to blocks such as:
    - Image block
    - Card block

See what happens:

https://github.com/gambitph/Stackable/assets/103395655/d6196fa2-a017-4ce8-988c-fa7f8be7529a

**Describe the solution you'd like**
Should be able to include alt text when fetching Dynamic Content image data.

**Describe alternatives you've considered**
N/A
